examples/rbd/plugin-deploy.sh

```diff
@@ -13,3 +15,15 @@ objects=(csi-provisioner-rbac csi-nodeplugin-rbac csi-config-map csi-rbdplugin-p
 for obj in "${objects[@]}"; do
 	kubectl create -f "./$obj.yaml"
 done
+
+if [[ -z $kms_base ]]; then
```

Comment:
new code is strongly encouraged to use the `${variable}` notation

Reply:
Sure. I'll change that. Thanks.